blueprints/networking/psc-hybrid/main.tf

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id         = var.project_create ? module.project.project_id : var.project_id
+  vpc_producer_id    = var.vpc_create ? module.vpc_producer.network.id : var.vpc_config["producer"]["id"]
+  vpc_producer_main  = var.vpc_create ? module.vpc_producer.subnets["${var.region}/${var.prefix}-main"].id : var.vpc_config["producer"]["subnet_main_id"]
+  vpc_producer_proxy = var.vpc_create ? module.vpc_producer.subnets_proxy_only["${var.region}/${var.prefix}-proxy"].id : var.vpc_config["producer"]["subnet_proxy_id"]
+  vpc_producer_psc   = var.vpc_create ? module.vpc_producer.subnets_psc["${var.region}/${var.prefix}-psc"].id : var.vpc_config["producer"]["subnet_psc_id"]
+  vpc_consumer_id    = var.vpc_create ? module.vpc_consumer.network.id : var.vpc_config["consumer"]["id"]
+  vpc_consumer_main  = var.vpc_create ? module.vpc_consumer.subnets["${var.region}/${var.prefix}-consumer"].id : var.vpc_config["consumer"]["subnet_main_id"]
+}
+
+module "project" {
+  count           = var.project_create ? 1 : 0
+  source          = "../../../modules/project"
+  billing_account = var.project_config["billing_account"]
+  name            = "psc-hybrid"
```

Comment:
name needs to be a variable and prefix `null` or it will be impossible to reference an existing project

it's a pattern we use widely across blueprints, can you please make sure to check repo best practices next time?

Reply:
Absolutely, sorry about that and thanks for double checking